exercises/custom-set/src/lib.rs

```diff
@@ -1 +1,61 @@
+use std::fmt::Debug;
+use std::marker::PhantomData;
 
+#[derive(Debug, PartialEq)]
+pub struct CustomSet<T> {
+    // This field is here to make the template compile and not to
+    // complain about unused type parameter 'T'. Once you start
+    // solving the exercise, delete this field and the 'std::marker::PhantomData'
+    // import.
+    phantom: PhantomData<T>,
+}
+
+impl<T: Debug> CustomSet<T> {
+    pub fn new(input: &[T]) -> Self {
+        unimplemented!(
+            "From the given input '{:?}' construct a new CustomSet struct.",
+            input
+        );
+    }
+
+    pub fn contains(&self, element: &T) -> bool {
+        unimplemented!(
+            "Determine if the '{:?}' element is present in the CustomSet struct.",
+            element
+        );
+    }
+
+    pub fn add(&mut self, element: T) {
+        unimplemented!("Add the '{:?}' element to the CustomSet struct.", element);
+    }
+
+    pub fn is_subset(&self, other: &Self) -> bool {
+        unimplemented!(
+            "Determine if the CustomSet struct is subset of the other '{:?}' struct.",
+            other
+        );
+    }
+
+    pub fn is_empty(&self) -> bool {
+        unimplemented!("Determine if the CustomSet struct is empty.");
+    }
+
+    pub fn is_disjoint(&self, other: &Self) -> bool {
+        unimplemented!(
+            "Determine if the CustomSet struct and othe struct '{:?}' are disjoint.",
```

Comment:
Spotted a typo `othe struct` should probably be `the other struct`. To reduce the noise, I would amend the fix to
the initial commit.

Reply:
Good spot! I missed that. Yes, please amend that into the single commit in this PR.